omi.js

```diff
@@ -287,8 +287,19 @@ var omi = {};
                 target: edge[1]
             });
 
-            result.nodes[edge[0]] = {};
-            result.nodes[edge[1]] = {};
+            tempNode = result.nodes[edge[0]];
+            if(!tempNode){
+                results.nodes[edge[0]] = [];

```

Comment:
Dit zou een object moeten zijn, als je edges er bij wilt opslaan, dan zou je dat in dit object onder een key kunnen zetten